parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -546,6 +563,19 @@ EOF
 # for DNS.
 iptables -I FORWARD -d 168.63.129.16 -p tcp --dport 80 -j DROP
 iptables -I FORWARD -d 168.63.129.16 -p tcp --dport 32526 -j DROP
+EOF
+
+    # As iptables rule will be cleaned every time the node is restarted, we need to ensure the rule is applied every time kubelet is started.
+    primaryNicIP=$(getPrimaryNicIP)
```

Comment:
I'd opt for using `logs_to_events` here instead just for the IMDS lookup - it will be accessible within this context

maybe something like `AKS.CSE.ensureKubelet.getPrimaryNicIP` would be a good event name

Reply:
updated code.